source/src_lcao/gint_tools.h

```diff
@@ -28,23 +28,13 @@ namespace Gint_Tools
 		const int jby,
 		const int kbz);
 
-	// index of wave functions for each block
-	// block_iw[na_grid]
-	int* get_block_iw(
-		const int na_grid,  		// how many atoms on this (i,j,k) grid
-		const int grid_index,		// 1d index of FFT index (i,j,k))
-		const int max_size);
-		
-	// block_index[na_grid+1]
-	int* get_block_index(
-		const int na_grid,  		// how many atoms on this (i,j,k) grid
-		const int grid_index);		// 1d index of FFT index (i,j,k)
-		
-	// band size: number of columns of a band
-	// block_size[na_grid]
-	int* get_block_size(
-		const int na_grid,			// how many atoms on this (i,j,k) grid
-		const int grid_index);		// 1d index of FFT index (i,j,k)
+	void get_block_info(
+		const int na_grid,
+		const int grid_index,
+		int * &block_iw,
+		int * &block_index,
+		int * &block_size
+	);
```

Comment:
You stripped comments out this function. Did you forget to add doc strings for your refactored functions?

Reply:
I'll add the comment back